### PR TITLE
run doc in python3

### PR DESCRIPTION
**Pull request by knorth55**
https://readthedocs.org/projects/knorth55-jsk-apc/builds/13695143/

### CODE REVIEW

**Comment by knorth55** [comment]
test fails because of https://github.com/jsk-ros-pkg/jsk_travis/issues/423

**Comment by knorth55** [comment]
weird test bug...
similar to this issue.
https://github.com/ros-infrastructure/rosdep/issues/807

**Comment by knorth55** [comment]
@pazeshun travis passed! can you review this PR?

File: doc/requirements.txt [hunk 1]
```diff
@@ -1,26 +1,5 @@
...
-Sphinx==1.3.3
-sphinx-autobuild==0.6.0
-sphinx-rtd-theme==0.1.9
-tornado==4.3
-watchdog==0.8.3
+recommonmark==0.5.0
+six==1.15.0
+Sphinx==3.3.0
+sphinx-markdown-tables==0.0.9
+sphinx_rtd_theme==0.4.3
```
Comment (pazeshun) [inline doc/requirements.txt:5]: Isn't this `sphinx-rtd-theme`?
cf. https://github.com/start-jsk/jsk_apc/pull/2737/files#diff-246970f36ae7ff82cc458ecc03e34c76cd67f7b50a0e4899b12c31c283b26fcfL24

Comment (knorth55) [reply doc/requirements.txt:5]: I think either is ok to install in pip.
But OK, I will use `sphinx-rtd-theme`.

Comment (pazeshun) [reply doc/requirements.txt]: Oh, sorry, I should have googled...

**Pull request merged**